### DISCUSSION TITLE
Problèmes dans les groupes et les pages

### DISCUSSION THREAD

**Discussion by GregoryCle** [requarks/wiki#7079] [Help / Questions]
Bonjour et merci d'avance pour votre aide.

Je e comprends pas comment fonctionne les groupes. Du moins, je n'arrive pas à faire fonctionner les accès aux répertoires comme je le souhaiterais.

Dans mon idéal,  je voudrais réaliser ceci:
![image](https://github.com/requarks/wiki/assets/73963018/1907a82b-336e-4e0f-bf3a-9d17fcd7c1fb)

Chaque personne qui se connecte, arrive sur la page <<Manuel d'utilisations>>

Selon le groupe, la personne peut voir les pages qui se trouvent dans ELE ou MEC.

J'ai donc créé deux groupes : USER_ELE et USER_MEC. Voici un exemple:
![image](https://github.com/requarks/wiki/assets/73963018/0ee4cf22-4303-4a66-ac9f-6b87bc6cf298)

Par la suite, j'ai associé les utilisateurs a un des groupes:
![image](https://github.com/requarks/wiki/assets/73963018/c05c40d3-a533-44cc-af1c-480776e13471)

Peu importe comme je fais mes groupes, mon utilisateur arrive à voir les autres pages. 
![image](https://github.com/requarks/wiki/assets/73963018/cbc48eb6-1fde-4851-a222-971771800e27)

En résumé, voici l'arbre idéal que j'aimerais construire:

Manuel d'utilisation=>
              Groupe ELE =>Pages ELE
              Groupe MEC =>Pages MES
              Groupe etc => Page etc

Seule les utilisateurs ELE peuvent voir les pages ELE, MEC peuvent voir les pages MEC, etc

Qu'est-ce que je ne fais pas de bien et comment faire ?

Merci

**Comment by NGPixel** [comment]
You need to set Page Rules in each group.

https://docs.requarks.io/groups

**Comment by GregoryCle** [comment]
Bonjour et merci de votre retour.

Cela ma effectivement aidé. Par contre, j'ai un autre souci.

Voici un exemple de structure : Energie/Batterie et Energie/Solaire

La page Batterie affiche bien l'article. Mais si je clique sur Energie, cela me dit que la page n'existe pas. Je ne veux pas de page pour Energie car cela est une catégorie.
![image](https://github.com/requarks/wiki/assets/73963018/09e0517d-a14a-424c-944f-9ecb6ebbfa4c)
![image](https://github.com/requarks/wiki/assets/73963018/67e8df99-c593-4306-9da2-7c4fcda03e35)

Je n'ai pas trouver comment faire pour éviter l'erreur.

Merci
